common.gradle

```diff
@@ -1,5 +1,7 @@
 versioning {
-    postProcessVersion = { version -> version.contains('-') ? "${version}-SNAPSHOT" : version }
+    postProcessVersion = { version ->
+        version.matches('^[^-]+(-alpha|-beta)(\\.\\d+)?[^-]*$') ? version : "${version}-SNAPSHOT"
```

Comment:
Was war an meinem Vorschlag nicht in Ordnung? Ich glaube jetzt matcht er auf normale Versionen nicht mehr und hängt da dann das -SNAPSHOT an.